src/mono/sample/HelloWorld/HelloWorld.csproj

```diff
@@ -35,4 +37,23 @@
       <Output TaskParameter="CompiledAssemblies" ItemName="BundleAssemblies" />
     </MonoAOTCompiler>
   </Target>
+
+  <UsingTask TaskName="ILStrip"
+             AssemblyFile="$(MonoTargetsTasksAssemblyPath)" />
+
+  <Target Name="StripILCode" Condition="'$(StripILCode)' == 'true'" AfterTargets="AOTCompileApp">
+    <PropertyGroup>
+      <TrimIndividualMethods>true</TrimIndividualMethods>
```

Comment:
I don't think it should be a default behaviour.
Something like: 
```xml
<TrimIndividualMethods Condition ="'$(MethodTokenFilePath)' != ''" >true</TrimIndividualMethods>
<TrimIndividualMethods Condition ="'$(TrimIndividualMethods)' == ''" >false</TrimIndividualMethods>
```
would be more suitable.

Reply:
`Condition="'$(StripILCode)' == 'true'"` has already server as the switch.

Reply:
The idea was just to make it more configurable, as passing `StripILCode=true` will wire `TrimIndividualMethods` to `true`.

Reply:
This example is to demonstrate how to trim away individual methods. I don't think it needs to be more configurable tho.